core/feed.go

```diff
@@ -779,12 +779,12 @@ func GetUserFeed(ctx context.Context, db *sql.DB, viewer *uid.ID, userID uid.ID,
 }
 
 func getCommentsPostTitles(ctx context.Context, db *sql.DB, comments []*Comment, viewer *uid.ID) error {
-	postToComment := make(map[uid.ID]*Comment)
+	postToComment := make(map[int]*Comment)
```

Comment:
Don't see why this variable would be necessary. It's a one-to-one mapping to the comments slice after the first loop.